spec/rails_helper.rb

```diff
@@ -67,6 +67,18 @@
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
 
+  config.before(:suite) do
```

Comment:
@josephelias-rubyians  This is something I would configure it as 

config.before type: :request do
    DatabaseCleaner.strategy = :truncation
  end

  config.before :suite do
    DatabaseCleaner.clean
  end

  config.before do
    DatabaseCleaner.start
  end

  config.after do
    DatabaseCleaner.clean
  end